core/src/com/mygdx/pirategame/GameScreen.java

```diff
@@ -157,6 +157,19 @@ public GameScreen(PirateGame game){
             Powerups.add(new Powerup(this, a, b, i));
         }
 
+        Tornadoes = new ArrayList<>();
+        for (int i = 0; i < 20; i++) {
```

Comment:
Does this need to be a hard-coded 20? On a small map this may mean that they'd be everywhere (and on a super small map a crash because of not enough free space?), and on a large one very rare...

Could move this to a param on the gameScreen and/or make them spawn/disappear mid-game...?

Maybe like...
```
on gameScreen...

// 1% chance of new one spawning on each update (same for delete)
public void update() {
...
if random(1,100) == 1 {
  Tornadoes.add(new Tornado(this, a, b));
}
...
}
```